starter-code/javascript/chronometer.js

```diff
@@ -1,23 +1,33 @@
 // Constructor
-// function Chronometer() {
+function Chronometer() {
+    this.currentTime = 0;
+    this.intervalId = null;
 
-// }
+}
 
-// Chronometer.prototype.startClick = function () {
+Chronometer.prototype.startClick = function () {
+    this.intervalId = window.setInterval(this.startClick,1000);
```

Comment:
What you should put on the first setInterval argument is the callback that you want to run every `1000`ms.
So you should have something like:
```
this.intervalId = window.setInterval(function () {
    this.currentTime++
  },1000);
```

Right now what you are doing is calling the method `startClick` every 1000ms. That means that you are creatin an interval each 1000ms since that is what you have in your line 9